default.nix

```diff
@@ -0,0 +1,24 @@
+{ nixpkgs ? import ./nixpkgs.nix
+, pkgs ? import nixpkgs {}
```

Comment:
As long as we don't pin, I would just go with `pkgs ? (import <nixpkgs> {})`, just as we do in shell.nix. Then you can remove the nixpkgs.nix file and skip line 1 here.